styles.css

```diff
@@ -342,3 +344,165 @@ form button:active {
   background-color: #2230d2;
   color: #fff;
 }
+
+/* desktop */
+
+@media (min-width: 769px) {
+  .lans {
+    margin: 1% 0 1% 0;
+  }
+
+  .names {
+    padding-bottom: 2%;
+  }
+
+  .lans .names {
+    float: right;
+  }
+
+  .lansIcons {
+    display: none;
+  }
+
+  .frontpage section {
+    background-image: none;
+    margin: 0 20%;
+  }
+
+  .b li {
+    margin-top: 3%;
+    padding: 1%;
+  }
+
+  .desktop2img {
+    float: right;
+    margin-right: 4%;
+    padding-left: 5%;
+  }
+
+  .desktop1img {
+    float: left;
+    margin-left: 4%;
+    padding-right: 5%;
+  }
+
+  .frontpage {
+    background-image: url(./Images/background/Header\ bg@2x.svg);
+    background-repeat: no-repeat;
+    background-size: 100%;
+    background-position: 50% 50%;
+    margin-bottom: 10%;
+  }
+
+  .desktopnav {
+    list-style: none;
+    display: flex;
+    gap: 24px;
+  }
+
+  .desktopnav a {
+    font-style: normal;
+    font-weight: 500;
+    font-size: 15px;
+    line-height: 20px;
+    color: #344563;
+  }
+
+  .desktop {
+    margin-right: 15%;
+  }
+
+  .mobile {
+    display: none;
+  }
+
+  header {
+    padding-top: 0.3%;
+  }
+
+  .logo {
+    font-style: normal;
+    font-weight: 700;
+    font-size: 18px;
+    line-height: 20px;
+    margin-left: 15%;
+  }
+
+  .frontpage p {
+    width: 100%;
+  }
+
+  .frontpage section h2 {
+    width: 40%;
+  }
+
+  .mobileimg {
+    display: none;
+  }
+
+  form button {
+    width: 10%;
+    margin: auto;
+    padding: 1%;
+  }
+
+  .proj1text button {
+    margin-left: 0;
+  }
+
+  .project1 button {
+    padding: 1%;
+    margin-top: 3%;
+    width: 10%;
+    border-radius: 10px;
+    border: 1px solid #396df2;
+    color: #396df2;
+    font-weight: 500;
+    font-size: 17px;
+    line-height: 24px;
+  }
```

Comment:
[OPTIONAL]
- [ ] When passe the cursor over your buttons the color of the text doesn't change (see image). You can fix that by removing the property `color: #396df2;` line 459 because it has been already specified in the Mobile CSS code.
![Screenshot from 2023-04-06 13-53-38](https://user-images.githubusercontent.com/91142432/230402916-7e44d3fa-a62c-48c0-b618-7724a3d711ec.png)